lib/content-services/src/lib/upload/components/file-uploading-dialog.component.ts

```diff
@@ -205,8 +207,6 @@ export class FileUploadingDialogComponent implements OnInit, OnDestroy {
         this.counterSubscription.unsubscribe();
         this.fileUploadSubscription.unsubscribe();
         this.errorSubscription.unsubscribe();
-        this.onDestroy$.next(true);
```

Comment:
I think you can remove also some of unsubscribing calls above from ngOnDestroy (and Subscription objects at all like listSubscription) because now unsubscribing is done by takeUntilDestroyed so some of them seems useless now